lib/Model/Suite.php

```diff
@@ -250,4 +258,41 @@ public function generateUuid()
             $this->configPath,
         ])), 0, -7);
     }
+
+    public function mergeBaselines(SuiteCollection $suiteCollection): self
+    {
+        foreach ($suiteCollection->getSuites() as $baselineSuite) {
+            foreach ($this->getVariants() as $variant) {
+                $subject = $variant->getSubject();
+                $benchmark = $subject->getBenchmark();
+
+                $baselineVariant = $baselineSuite->findVariant(
+                    $benchmark->getClass(),
+                    $subject->getName(),
+                    $variant->getParameterSet()->getName()
+                );
+
+                if (!$baselineVariant) {
+                    continue;
+                }
+
+                $variant->attachBaseline($baselineVariant);
+            }
+        }
```

Comment:
Multiple suites can be used to provide baselines, in case of multiple matching variants, the last matching variant wins.